src/encoded/tests/test_audit_experiment.py

```diff
@@ -128,14 +128,71 @@ def test_ChIP_possible_control_roadmap(testapp, base_experiment, ctrl_experiment
                for error in collect_audit_errors(res))
 
 
-def test_audit_input_control(testapp, base_experiment,
-                             ctrl_experiment, IgG_ctrl_rep):
-    testapp.patch_json(ctrl_experiment['@id'], {'control_type': 'control'})
-    testapp.patch_json(base_experiment['@id'], {'possible_controls': [ctrl_experiment['@id']],
-                                                'assay_term_name': 'ChIP-seq'})
+def test_audit_input_control(
+    testapp,
```

Comment:
it's not easy to follow what cases are being tested here, I think you have all of them though.